interfaces/zookeeper/v0/schema.py

```diff
@@ -0,0 +1,94 @@
+"""This file defines the schemas for the provider and requirer sides of the zookeeper_client interface.
+
+It must expose two interfaces.schema_base.DataBagSchema subclasses called:
+- ProviderSchema
+- RequirerSchema
+"""
+
+from typing import TypeAlias
+
+from interface_tester.schema_base import DataBagSchema
+from pydantic import (
+    BaseModel,
+    Field,
+    IPvAnyAddress,
+    conint,
+)
+
+
+class Endpoint(BaseModel):
```

Comment:
I don't see this class being used anywhere, so perhaps can be removed?

Reply:
Yeah, that was a remnant of a previous more complex validation step for the databag. Removed in 8687e858332bbbbab6041c4cfbfd48136fd11b9f